ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -320,5 +320,12 @@
     "CaseEventID": "addStatementOfService",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createNoticeOfProceedings",
+    "UserRole": "caseworker-publiclaw-courtadmin",
```

Comment:
Do we need to enable permissions for cafcass also? Probably not if we are going to remove permissions again once merged.

Reply:
I've re-enabled permissions for cafcass anyway; so now admins and cafcass can create C6